### DISCUSSION TITLE
Topic similarity

### DISCUSSION THREAD

**Discussion by MolkaSlama** [MaartenGr/BERTopic#1476] [Q&A]
I know that for modelling LDA subjects it is more accurate to calculate similarity using the Hellinger distance. Is this the case for bertopic or is it more useful to calculate similarity using cosine similarity??

**Comment by MaartenGr** [comment]
There is not one similarity metric that *should* be used over another as it depends on how you approach the problem. Generally though, topic similarity in BERTopic will be calculated through the c-TF-IDF or topic embedding representations which generally work well with cosine similarity. Having said that, especially with c-TF-IDF, using different metrics like the Hellinger distance might be worthwhile.

**Reply by MolkaSlama** [reply]
Thank you for your answer

**Reply by MolkaSlama** [reply]
How can i get the c-TF-IDF or the topic embedding ?

**Reply by MaartenGr** [reply]
You can find BERTopic's attributes here: https://github.com/MaartenGr/BERTopic#attributes
